jbi/runner.py

```diff
@@ -38,23 +39,26 @@ def execute_action(
         )
 
         try:
-            bug_obj: BugzillaBug = request.bugzilla_object
+            if request.bug.is_private:
+                request = request.copy(
+                    update={"bug": get_bugzilla().get_bug(request.bug.id)}
```

Comment:
This pattern seems handy enough that maybe the method you added to `LogContext` should be added to a `BaseModel` that all of our models. Not requesting a change here, just thinking out loud.